Setup/Patch/Data/DefaultConfigs.php

```diff
@@ -12,6 +12,7 @@
 use Wizzy\Search\Services\Setup\Version135;
 use Wizzy\Search\Services\Setup\Version1316;
 use Wizzy\Search\Services\Setup\Version1317;
+use Wizzy\Search\Services\Setup\Version223;
 
 class DefaultConfigs implements DataPatchInterface
```

Comment:
Along with adding version to default configs. We also need to create new patch where it is executed for already installed instances.